packages/plugins/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@webiny/plugins",
   "version": "3.0.1",
-  "main": "dist/index.js",
+  "main": "index.js",
```

Comment:
For future testing, it's easier to simply import the packages with the `dist` path included, like: 
```import { PluginsContainer } from "@webiny/plugins/dist"```.
This is only necessary when working on the `cli` package.